src/modules/commander/HealthAndArmingChecks/HealthAndArmingChecks.cpp

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2020 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2022 PX4 Development Team. All rights reserved.
```

Reply:
Believe it or not, this is when it was actually created